src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,26 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin desktop {
+  @media (min-width: 1024px) {
+    @content;
+  }
+}
+
+@mixin tablet {
+  @media (min-width: 740px) {
+    @content;
+  }
+}
+
+@mixin top-actions {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__utils {
+    display: flex;
+  }
+
+}
```

Comment:
```suggestion
  }
}
```

Reply:
sorry

Reply:
no need to sorry!)
I've left the comments that you can improve yourself and learn smth new or just improve it if you knew it and forgot 😉